docs/source/getting_started/basic_usage.rst

```diff
@@ -0,0 +1,115 @@
+Basic project commands
+##############################################################################
+
+This page covers basic Django and system commands that both authors and
+developers will use when using the CS Unplugged system.
+
+.. note::
+
+  We assume by this point you have installed the project, checked the
+  setup is working (see :ref:`installation-check-project-setup-works`),
+  and also have a basic understanding of the
+  :doc:`project structure <project_structure>`.
+
+These commands can be run from terminal from within the ``csunplugged`` folder
+(note: this is the ``cs-unplugged/csunplugged`` folder).
+When working on the project, you should type the following commands in order
+to preview the work you have done.
+
+.. note::
+
+  The following commands must be run with Python 3. If you followed our
+  installation guide, we have set ``python`` to refer to Python 3 already.
+  Otherwise you will need to use ``python3`` instead of ``python``.
```

Comment:
should make sure to emphasise that this is only the case if you choose to set up a `venv`, otherwise this might not be true